go.mod

```diff
@@ -50,6 +50,7 @@ require (
 	golang.org/x/lint v0.0.0-20210508222113-6edffad5e616
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/sys v0.0.0-20220715151400-c0bba94af5f8
+	golang.org/x/time v0.0.0-20210723032227-1f47c861a9ac
```

Comment:
This commit is behind the head of the module: https://cs.opensource.google/go/x/time;bpv=1

Ping this to `v0.3.0` which is also the latest commit in the package.